actors/miner/tests/prove_commit.rs

```diff
@@ -25,397 +16,8 @@ mod util;
-    );
-
-    // Prove just the first sector, with no deals
-    {
-        let precommit = &precommits[0];
-        let sector = h
-            .prove_commit_sector_and_confirm(
-                &rt,
-                precommit,
-                h.make_prove_commit_params(precommit.info.sector_number),
-                ProveCommitConfig::default(),
-            )
-            .unwrap();
-        assert_eq!(*rt.epoch.borrow(), sector.activation);
-        let st = h.get_state(&rt);
-        let expected_deposits = 2 * pre_commit_deposit_for_power(
-            &h.epoch_reward_smooth,
-            &h.epoch_qa_power_smooth,
-            &qa_power_max(h.sector_size),
-        ); // first sector deposit released
-
-        assert_eq!(expected_deposits, st.pre_commit_deposits);
-
-        // Expect power/pledge for a sector with no deals
-        assert_eq!(no_deal_pledge, sector.initial_pledge);
-        assert_eq!(no_deal_pledge, st.initial_pledge);
-    }
-    // Prove the next, with one deal
-    {
-        let precommit = &precommits[1];
-        let mut pcc = ProveCommitConfig::empty();
-        pcc.add_activated_deals(precommit.info.sector_number, vec![verified_deal1]);
-        let sector = h
-            .prove_commit_sector_and_confirm(
-                &rt,
-                precommit,
-                h.make_prove_commit_params(precommit.info.sector_number),
-                pcc,
-            )
-            .unwrap();
-        assert_eq!(*rt.epoch.borrow(), sector.activation);
-        let st = h.get_state(&rt);
-        let expected_deposits = pre_commit_deposit_for_power(
-            &h.epoch_reward_smooth,
-            &h.epoch_qa_power_smooth,
-            &qa_power_max(h.sector_size),
-        ); // first and second deposit released
-
-        assert_eq!(expected_deposits, st.pre_commit_deposits);
-
-        // Expect power/pledge for the two sectors (only this one having any deal weight)
-        assert_eq!(full_deal_pledge, sector.initial_pledge);
-        assert_eq!(&no_deal_pledge + &full_deal_pledge, st.initial_pledge);
-    }
-    // Prove the last
-    {
-        let precommit = &precommits[2];
-        let mut pcc = ProveCommitConfig::empty();
-        pcc.add_activated_deals(precommit.info.sector_number, vec![verified_deal2, verified_deal3]);
-        let sector = h
-            .prove_commit_sector_and_confirm(
-                &rt,
-                precommit,
-                h.make_prove_commit_params(precommit.info.sector_number),
-                pcc,
-            )
-            .unwrap();
-        assert_eq!(*rt.epoch.borrow(), sector.activation);
-        let st = h.get_state(&rt);
-        assert!(st.pre_commit_deposits.is_zero());
-
-        // Expect power/pledge for the three sectors
-        assert_eq!(&full_deal_pledge, &sector.initial_pledge);
-        assert_eq!(&no_deal_pledge + &full_deal_pledge + &full_deal_pledge, st.initial_pledge);
-    }
-}
-
-#[test]
-fn invalid_proof_rejected() {
```

Comment:
@anorth I'm more sure about the other tests but could you confirm that we are not throwing away any valuable test coverage here for prove_commit3?

Reply:
With existing tests in prove_commit2_failures_test.rs plus #1547 I think we're good.